streams/ng.m3u

```diff
@@ -49,7 +51,7 @@ https://webstreaming.viewmedia.tv/web_011/Stream/playlist.m3u8
 https://webstreaming-11.viewmedia.tv/web_152/Stream/playlist.m3u8
 #EXTINF:-1 tvg-id="LN247.ng",LN247 (1080p)
 https://go5lmb6oyawb-hls-live.5centscdn.com/station/3dfd3752af3d7aec5c53992c2da3a316.sdp/playlist.m3u8
-#EXTINF:-1 tvg-id="",Lovetoons TV (720p)
+#EXTINF:-1 tvg-id="LovetoonsTV.ng",Lovetoons TV (720p)
 https://kali.vdopanel.com:3674/stream/play.m3u8
```

Comment:
Not loading for me. Geo-locked, not-24/7 or down.

Reply:
https://kali1.everestcast.com:3674/stream/play.m3u8 will have no issues with https. First links also works for me if certificate issue is ignored
```suggestion
https://kali1.everestcast.com:3674/stream/play.m3u8
```